contracts/Comptroller.sol

```diff
@@ -1019,6 +1030,40 @@ contract Comptroller is ComptrollerV3Storage, ComptrollerInterface, ComptrollerE
             require(allMarkets[i] != CToken(cToken), "market already added");
         }
         allMarkets.push(CToken(cToken));
+
+        // Initially, set borrow limit to unlimited. This should be set appropriately in the same governance call immediatly after adding the market.
+        _setMarketBorrowLimitInternal(CToken(cToken),uint(-1));
+
+    }
+
+
+    /**
+      * @notice Set the given borrow limit for the given cToken market
+      * @dev Admin function to set the borrow limit
+      * @param cToken The address of the market (token) to change the borrow limit for
+      * @param borrowLimit The new borrow limit value in underlying (maximum borrowing) to be set
+      * @return uint 0=success, otherwise a failure. (See enum Error for details)
+      */
+    function _setMarketBorrowLimit(CToken cToken, uint256 borrowLimit) public returns (uint) {
+        // Allow pause guardian to change borrow limit too.
+        if (msg.sender != admin && msg.sender != pauseGuardian) {
+            return fail(Error.UNAUTHORIZED, FailureInfo.SET_MARKET_BORROW_LIMIT_OWNER_CHECK);
+        }
+
+        _setMarketBorrowLimitInternal(cToken,borrowLimit);
+
+        return uint(Error.NO_ERROR);
+    }
+
+    function _setMarketBorrowLimitInternal(CToken cToken, uint256 borrowLimit) internal {
+        borrowLimits[address(cToken)] = borrowLimit;
+
+        emit NewBorrowLimit(cToken,borrowLimit);
```

Comment:
Can you ensure there's a comma after all arg lists?

Reply:
What do you mean by this?

Reply:
Just meant the missing space